### ISSUE TITLE
Module 4: Displayif syntax

### ISSUE THREAD

**Issue by meekk1** [episphere/quest#138]
![image](https://user-images.githubusercontent.com/64271614/85293707-b8042080-b46b-11ea-86df-2b4f771ba8fc.png)

**Comment by danielruss** [comment]
I am adding Alaina because I am interpreting the questions differently that what is written, and Alaina may correct me.

Since the questionnaire is not asking a question with a set of follow up questions, we cannot use my preferred syntax with arrows.

The variables WORK, WORK6, and STUDENT come from Module1.

So I think, for all question from COMINTRO to CURWORKT14 (note T14 not T4),  you want to add the displayif=equals(WORK,1). These will display if and only if work=1.

WORK6 [Have you ever worked for pay] would only be asked if WORK=0.  
So the logic should just be :  [ID,displayif=equals(WORK6,1)].   
This logic should be applied to questions PREWORKRES-PREREST8 

Finally: the logic ID,displayif=or(equals(STUDENT,1),equals(STUDENT,2)] should be applied to INTROCURSCH to the end of the module.

In conclusion,  
Question COMINTRO to CURWORKT14 are only asked if WORK=1
Question PREWORKRES to PREREST9 are only asked if WORK6=1 [and WORK=0]
Question INTROCURSCH to CURSCHT14 are only asked if STUDENT=1 or STUDENT=2

**Comment by ashreves** [comment]
@danielruss I believe your explanation of the syntax is accurate.

**Comment by danielruss** [comment]
@meekk1 did I answer your question?

**Comment by meekk1** [comment]
I think so. Still working through it. I'll close this and open it back up if I have any other questions. Thanks.
